plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -1915,3 +1915,14 @@ SecRule REQUEST_FILENAME "@rx /apps/mail/api/outbox/(?:from-draft/)?[0-9]+$" \
     ctl:ruleRemoveTargetByTag=OWASP_CRS;ARGS:json.body,\
     ctl:ruleRemoveTargetByTag=OWASP_CRS;ARGS:json.editorBody,\
     ctl:ruleRemoveTargetByTag=OWASP_CRS;ARGS:json.subject"
+
+# Moving a card in Deck application
+SecRule REQUEST_FILENAME "@rx /apps/deck/cards/[0-9]+/reorder?$" \
```

Comment:
The question mark at the end makes only the `r` optional, which is probably not your intention.